sagemaker-script-mode/sklearn/sklearn_byom.ipynb

```diff
@@ -2,27 +2,37 @@
  "cells": [
```

Comment:
<p>Small style note - Use present tense rather than future tense throughout. I.e. "the model will be trained" -&gt; "the model is trained"</p><p>From the contrib guide:</p><blockquote>Use present tense.</blockquote><blockquote>👍&nbsp;"The estimator fits a model."</blockquote><blockquote>👎&nbsp;"The estimator will fit a model."</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>